python/taichi/_main.py

```diff
@@ -188,31 +194,30 @@ def draw_bounding_box(rind, cind):
         def on_mouse_click_callback(example_name):
             examples_dir = TaichiMain._get_examples_dir()
             script = list(examples_dir.rglob(f"{example_name}.py"))[0]
-            with open(script, "r") as f:
-                try:
-                    import rich.console  # pylint: disable=C0415
-                    import rich.syntax  # pylint: disable=C0415
-                    content = rich.syntax.Syntax.from_path(script,
-                                                           line_numbers=True)
-                    console = rich.console.Console()
-                    console.print(content)
-                except ImportError:
-                    content = f.readlines()
-                    print(content)
+            try:
```

Comment:
```suggestion
            print("Demo source code:")
            print()
            try:
```

Let's add a prompt here.